examples/graphql-file-upload/graphql_file_upload.metatags

```diff
@@ -1,2 +1,2 @@
-description: A GraphQL service endpoint written in Ballerina
+description: This example demonstrates uploading files to a GraphQL schema.
```

Comment:
```suggestion
description: This example demonstrates implementing GraphQL service with file uploading capability.
```

Reply:
Updated as: `This example demonstrates implementing a GraphQL service with the file uploading capability.`.